add skystone and skystone dust recipes

Rafmaninoff opened the pull request:
Add crafting recipes for skystone for decoration or to facilitate access to the material for users that have channels enabled. 
skystone is endstone + stone in the alloy smelter, also adds skystone dust.

ALongStringOfNumbers commented:
Could you give some reasoning behind the EU/t amounts that you chose?

Rafmaninoff commented:
Hi, I Didn't put too much thought into the EU/t ammounts, I just wanted to make the recipes be MV-tier since AE is also unlocked at MV.

I'm absolutely open to changing it, if you have some sort of standard going.

ALongStringOfNumbers commented:
There was some discussion after this popped up, and it was determined that it would be fine if the recipes were LV recipes.  For the macerator recipe, it would probably be fine to follow the obsidian dust recipe (8 EU/t for 1.5s), and for the Alloy smelter recipe, something at 16 EU/t would be fine. 

Additionally, please move your additions from your new file into the existing AE2.zs file.

Rafmaninoff commented:
I now merged the recipes into the main file and adjusted them. for the duration of the alloy smelter I went with the standard for LV alloys like steel and bronze, 16EU/t for 10s.

(yes, the commits are me failing at git, no bully.)

ALongStringOfNumbers commented:
Please update to resolve merge conflicts.

Exaxxion committed:
PR Cleanup

Revert all prior commits, rebase to origin/dev, add just the substantive
change at the end of AE2.zs as it appeared in 1a6a3b0.

Exaxxion commented:
I would ask that in general we do not modify lines unrelated to actual changes so as to preserve the relevant commit history and make it clear what is actually being done.

As such I have coalesced your commits into a single one leaving just the two lines of actual changes as they appeared at the end of `AE2.zs`.

Exaxxion commented:
The repository hosting commits for this pull request was destroyed by the owner. I am going to close this PR and move the proposed changes to a local feature branch so we can finish the review.
